tron/actioncommand.py

```diff
@@ -122,31 +112,15 @@ def is_failed(self):
     @property
     def is_complete(self):
         """Complete implies done and success."""
-        return self.machine.state == self.COMPLETE
+        return self.machine.state == 'complete'
```

Comment:
Why are these and and :120 the odd ones out?  These are still part of the same object, why not use the constants?

Reply:
yep, that's a mistake, fixing